projects/solar/test/test_data_store.c

```diff
@@ -6,47 +6,85 @@
 #include "test_helpers.h"
 #include "unity.h"
 
+#define VALID_TEST_DATA_POINT DATA_POINT_VOLTAGE_1
+#define VALID_TEST_VALUE 0x1337
+
 void setup_test(void) {
   event_queue_init();
 }
 void teardown_test(void) {}
 
 // Test that we can set and get a value from the data store.
 void test_data_store_set_and_get_basic(void) {
+  TEST_ASSERT_OK(data_store_init());
```

Comment:
Is there a reason you don't just add `TEST_ASSERT_OK(data_store_init())` to `setup_test()`? I dunno if that's not best practice but it workes, I'm fine with either

Reply:
Probably something weird would happen if an assertion failed in `setup_test`, that's why I didn't put it there